index.js

```diff
@@ -50,11 +57,20 @@ class APCAccess {
     this.batteryService
       .getCharacteristic(Characteristic.StatusLowBattery)
       .on('get', this.getStatusLowBattery.bind(this));
+
+    this.temperatureService = new Service.TemperatureSensor();
+    this.temperatureService
+      .getCharacteristic(Characteristic.CurrentTemperature)
+      .on('get', this.getTemperature.bind(this));
```

Comment:
If we have the temp service turned off or not available, then it would be better not to add it into HK, as it will just add a temp sensor to the room that never receives data.

Reply:
Also you probably don't want to be calling`getTemperature()` if the UPS doesn't support a temp sensor, as your'll just end up logging lots of errors.

Reply:
This should be gated by the publication here, so none of that is invoked: https://github.com/cr3ative/homebridge-apcaccess/blob/af3915d658cf200bf1eb71e8be0d6c9fc9849d0e/index.js#L71